src/js/style.js

```diff
@@ -4,7 +4,8 @@ import * as Layers from "../layer/index.js";
 export function build(tileURL, spriteURL, locales) {
   return {
     name: "Americana",
-    glyphs: "fonts/{fontstack}/{range}.pbf",
+    glyphs:
+      "https://zelonewolf.github.io/openstreetmap-americana/fonts/{fontstack}/{range}.pbf",
```

Comment:
If you want to make the fonts host-neutral, then the function would need to take the glyph URL as a parameter, just like it takes the tile and spritesheet URLs. This parameterization only benefits the command-line style JSON generation tool anyhow.